src/modules/homeView.js

```diff
@@ -0,0 +1,144 @@
+import { likeRocket, getLikes, getComments } from './apis.js';
+import { popupView } from './commentPopup.js';
+
+const homeContainer = document.querySelector('#home-container');
+
+// render data of the homepage
+const homepageView = async (data) => {
+  const rocketsContainer = document.querySelector('.rockets-container');
+  let rockets = '';
+
+  try {
+    const likesData = await getLikes(); // Retrieve the likes data from the API
+    const likesMap = new Map();
+
+    /* eslint-disable camelcase */
```

Comment:
- [ ] Please avoid disabling linter rules as they serve as a way to provide a consistent code format and ensure you use best practices. Instead, consider fixing the errors! 🤓 🔧